tee-worker/docker/lit-test-stress-script.yml

```diff
@@ -0,0 +1,25 @@
+services:
+    lit-test-stress-script:
+        image: litentry-cli:dev
+        container_name: litentry-test-stress-script
+        volumes:
+            - ../ts-tests:/ts-tests
+            - ../cli:/usr/local/worker-cli
+        build:
+            context: ..
+            dockerfile: build.Dockerfile
+            target: deployed-client
+        depends_on:
+            litentry-node:
+                condition: service_healthy
+            litentry-worker-1:
+                condition: service_healthy
+            litentry-worker-2:
+                condition: service_healthy
```

Comment:
TODO: should probably drop this one? /cc @Kailai-Wang @felixfaisal

Reply:
yea if you are only sending requests to worker-1 you don't have to wait for worker-2 (it's still launched though)